### ISSUE TITLE
object names must not include packages

### ISSUE THREAD

**Issue by yegor256** [objectionary/jeo-maven-plugin#271]
This is my Java:

```
package foo.bar;
class Test {}
```

This is the EO I'm getting in `generated-sources/eo/foo/bar`:

```
[] > foo/bar/Test
  ...
```

Instead, it should be getting:

```
+package foo.bar
[] > Test
  ...
```

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#284](https://github.com/objectionary/jeo-maven-plugin/issues/284), [#285](https://github.com/objectionary/jeo-maven-plugin/issues/285) are still not solved.

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#285](https://github.com/objectionary/jeo-maven-plugin/issues/285) is still not solved; solved: [#284](https://github.com/objectionary/jeo-maven-plugin/issues/284).

**Comment by 0pdd** [comment]
@yegor256 all 2 puzzles are solved here: [#284](https://github.com/objectionary/jeo-maven-plugin/issues/284), [#285](https://github.com/objectionary/jeo-maven-plugin/issues/285).
